CIPs/cip-draft_left-right_prf.md

```diff
@@ -0,0 +1,88 @@
+---
+cip: <to be assigned>
```

Comment:
is this CIP the same as the other one?

Reply:
Yes. I accidentally copied the file instead of renaming it. It should only have one now.